cuda/base/cusparse_bindings.hpp

```diff
@@ -160,15 +166,15 @@ GKO_BIND_CUSPARSE_TRANSPOSE64(ValueType, detail::not_implemented);
         const ValueType *OrigValA, const int32 *OrigRowPtrA,                 \
         const int32 *OrigColIndA, ValueType *TransValA, int32 *TransRowPtrA, \
         int32 *TransColIndA, cusparseAction_t copyValues,                    \
-        cusparseIndexBase_t idxBase) GKO_NOT_IMPLEMENTED;
+        cusparseIndexBase_t idxBase) GKO_NOT_IMPLEMENTED
 
 #define GKO_BIND_CUSPARSE_CONJ_TRANSPOSE64(ValueType, CusparseName)          \
     inline void conj_transpose(                                              \
         cusparseHandle_t handle, size_type m, size_type n, size_type nnz,    \
         const ValueType *OrigValA, const int64 *OrigRowPtrA,                 \
         const int64 *OrigColIndA, ValueType *TransValA, int64 *TransRowPtrA, \
         int64 *TransColIndA, cusparseAction_t copyValues,                    \
-        cusparseIndexBase_t idxBase) GKO_NOT_IMPLEMENTED;
+        cusparseIndexBase_t idxBase) GKO_NOT_IMPLEMENTED
```

Comment:
Do we need to remove the semicolons after `GKO_NOT_IMPLEMENTED` after you also added the `static_asserts` there?

Reply:
Good point, in fact they should be required.